lib/js/components/TabItem/TabItem.vue

```diff
@@ -1,40 +1,85 @@
 <template>
-	<div class="a-tabItem" :title="title" :class="{ '-isActive': isActive }">
-		<wnl-icon :icon="icon" :size="ICON_SIZES.X_SMALL" />
+	<div
+		class="tabItem"
+		:title="label"
+		:class="{
+			'-sizeMedium': size === TAB_ITEM_SIZES.MEDIUM,
+			'-sizeSmall': size === TAB_ITEM_SIZES.SMALL,
+			'-isSelected': isSelected,
+		}"
+		@click="$emit('click')"
+	>
+		<wnl-icon class="tabItem__icon" :icon="icon" :size="iconSize" />
+		<span class="tabItem__label">{{ label }}</span>
 	</div>
 </template>
 
 <style scoped lang="scss">
-@import '../../../styles/settings/icons';
 @import '../../../styles/settings/spacings';
 @import '../../../styles/settings/colors/tokens';
+@import '../../../styles/settings/typography/tokens';
 
-$tab-item-width: (2 * $space-s) + $icon-xs;
+.tabItem {
+	$self: &;
 
-.a-tabItem {
+	align-items: center;
 	border-bottom: 1px solid $color-neutral-border;
-	color: $color-neutral-icon;
 	cursor: pointer;
-	display: flex;
+	display: inline-flex;
 	justify-content: center;
-	padding: $space-xxs $space-s;
-	width: $tab-item-width;
+
+	&__icon {
+		color: $color-neutral-icon;
+	}
+
+	&__label {
+		color: $color-neutral-text;
+	}
 
 	&:hover {
 		border-bottom-color: $color-default-border;
-		color: $color-default-icon;
+
+		#{$self}__icon {
+			color: $color-default-icon;
+		}
+
+		#{$self}__label {
+			color: $color-default-text;
```

Comment:
maybe we could add a transition on these color change?

Reply:
something like `transition: color ease-in-out $default-transition-time;`?

Reply:
yes, similar to `border-bottom-color`